sdk/appplatform/Azure.ResourceManager.AppPlatform/src/Generated/Models/Error.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+namespace Azure.ResourceManager.AppPlatform.Models
+{
+    /// <summary> The error code compose of code and message. </summary>
+    public partial class Error
```

Comment:
Single word class name, should rename it.

Reply:
rename to `AppPlatformErrorInfo`